### ISSUE TITLE
Extremely low accuracy on UCF with pre-trained model

### ISSUE THREAD

**Issue by chengengliu** [okankop/Efficient-3DCNNs#41]
Hi there,
thanks for the work. We are trying to reproduce the experiments, however, both training and validation top1 accuracy were closed to 0. We followed the configuration in another issue #3 , with `batch_size` adjusted:
`python main.py --root_path ~/
--video_path ~/datasets/jester
--annotation_path Efficient-3DCNNs/annotation_UCF101/ucf101_01.json
--result_path Efficient-3DCNNs/results
--pretrain_path Efficient-3DCNNs/results/kinetics_shufflenet_0.5x_G3_RGB_16_best.pth
--dataset ucf101
--n_classes 600
--n_finetune_classes 101
--ft_portion last_layer
--model shufflenet
--groups 3
--width_mult 0.5
--train_crop random
--learning_rate 0.1
--sample_duration 16
--batch_size 64
--n_threads 16
--checkpoint 1
--n_val_samples 1 \` Anyone could provide any thoughts? Thanks
